hrms/hr/doctype/attendance_request/attendance_request.py

```diff
@@ -22,12 +22,26 @@ def validate(self):
 		validate_dates(self, self.from_date, self.to_date, False)
 		self.validate_half_day()
 		self.validate_request_overlap()
+		self.validate_no_attendance_to_create()
 
 	def validate_half_day(self):
 		if self.half_day:
 			if not getdate(self.from_date) <= getdate(self.half_day_date) <= getdate(self.to_date):
 				frappe.throw(_("Half day date should be in between from date and to date"))
 
+	def validate_no_attendance_to_create(self):
+		attendance_warnings = self.get_attendance_warnings()
+		attendance_request_days = date_diff(self.to_date, self.from_date) + 1
+		if len(attendance_warnings) == attendance_request_days and not any(
+			warning["action"] == "Overwrite" for warning in attendance_warnings
+		):
+			frappe.throw(
+				title=_("No attendance records to create"),
+				msg=_(
+					"Please check if employee is on leave or attendance with the same status exists for selected day(s)."
```

Comment:
Incorrect sentence framing? Also, you can link the existing attendance records in the message itself instead of asking the user to find them?